scipy/spatial/tests/test_distance.py

```diff
@@ -247,7 +546,7 @@ def dummy_metric(x, y):
         right_y = 1.123
         assert_equal(cdist_y, right_y, verbose=verbose > 2)
 
-    def _check_calling_conventions(self, X1, X2, metric, eps=1e-07, **kwargs):
+    def _check_calling_conventions(self, X1, X2, metric, eo_name, eps=1e-04, **kwargs):
```

Comment:
yes, there were a few functions based on p-norm where simplifying the python implementation slightly changed the values.